### ISSUE TITLE
Fonts from google are getting blocked if requested from http when PdnsAd. is run via https

### ISSUE THREAD

**Issue by piwats** [ngoduykhanh/PowerDNS-Admin#35]
When running PowerDNS-Admin via https, and using Chrome, one gets the following error:

```
Mixed Content: The page at 'https://dnsadmin.net/pdnsadmin/domain/example.com' was loaded over HTTPS, but requested an insecure stylesheet 'http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700&subset=all'. This request has been blocked; the content must be served over HTTPS.
```

That is because of the links in the head:
```html
<link href="http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700&subset=all" rel="stylesheet" type="text/css"/>
 <link href='http://fonts.googleapis.com/css?family=Roboto+Mono:400,300,700' rel='stylesheet' type='text/css'>
```

The files in question are:

app/templates/base.html
app/templates/login.html

Related: #30

**Comment by TacoScheltema** [comment]
I believe the proper syntax should be
```
<link href="//fonts.googleapis.com/css?family=Open+Sans:400,300,600,700&subset=all" rel="stylesheet" type="text/css"/>
```
That way the browser will use the same protocol as it used to load the page itself.

**Comment by ngoduykhanh** [comment]
Yes, It should be `//` instead of `http://` in the url. Fixed it commit bca3005
